components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -2267,24 +2267,42 @@ def _annotations(request, objtype, objid, conn=None, **kwargs):
     query += """
```

Comment:
@will-moore without extra ```join fetch links.details.owner``` this query will not load owner details when working in none default groups, which results in exception later on in the code.